xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -6284,8 +6284,8 @@ There is a difference in the behavior of the `GetLastWin32Error` method on .NET
       <Docs>
         <param name="ptr">The address of the first character of the unmanaged string.</param>
         <param name="len">The byte count of the input string to copy.</param>
-        <summary>Allocates a managed <see cref="T:System.String" />, copies a specified number of characters from an unmanaged ANSI string into it, and widens each ANSI character to Unicode.</summary>
-        <returns>A managed string that holds a copy of the native ANSI string if the value of the <paramref name="ptr" /> parameter is not <see langword="null" />; otherwise, this method returns <see langword="null" />.</returns>
+        <summary>Allocates a managed <see cref="T:System.String" />, copies a specified number of characters from an unmanaged ANSI or UTF-8 string into it, and widens each character to Unicode.</summary>
```

Comment:
Can we change the mention of Unicode to UTF-16? (Both mentions)

Reply:
Makes full sense. We have a lot of "Unicode" to change in this area.